ruby.test/syntax/lexical/identifier/IdentifierReservedKeywords.spt

```diff
@@ -0,0 +1,49 @@
+module IdentifierReservedKeywords
+
+language Ruby
+
+start symbol ID
+
+// Automatically generated from doc/reserved-keywords.txt
```

Comment:
Should this not (also) be stated in the actual Identifier definition perhaps? Because it is automatically generated it would nice to document this in some sort of convenient way.